components/playlist/browser/resources/utils/bytesFormatter.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { PlaylistItem } from 'gen/brave/components/playlist/common/mojom/playlist.mojom.m.js'
+
+// Note that we hardly expect files to be bigger than 1TB. Generally, 4k 1 hour long
+// video is 30GB more ore less. So it's unlikely for the |bytes| to exceed
+// the number range. Roughly we can represents more than 10000 videos in that
+// size with number type.
+// * Number.MAX_SAFE_INTEGER == 9007199254740991
+// *  30 GB                  == 30000000000
+export function formatBytes (bytes: bigint) {
+  // Here, we convert the bytes into number first so that we can
+  // get digits in decimal places.
+  const mb = Number(bytes) / 1_000_000
```

Comment:
I think you can also use the `1e6` notation, if you prefer it :smile:

Reply:
Also, slightly controversial and unimportant question, but we're going with base `1000` rather than `1024`?

Reply:
I don't really have any strong feelings :laughing: I remember getting so confused by it :laughing: I think either way will be fine :smile:

Reply:
It looks like DownloadItem from upstream uses GiB unit for the UI

https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:ui/base/text/bytes_formatting.cc;l=72;drc=4763cf47e6f4919821efa6bc1ca3c7757125b54c;bpv=1;bpt=1

Let's use units with power of 2